src/TxHistory/TxHistoryNavigator.tsx

```diff
@@ -31,23 +32,22 @@ const Stack = createStackNavigator<TxHistoryRoutes>()
 export const TxHistoryNavigator = () => {
   const strings = useStrings()
   const wallet = useSelectedWallet()
+
   const walletName = useWalletName(wallet)
   const transactionInfos = useSelector(transactionsInfoSelector)
-  const tokenBalance = useSelector(tokenBalanceSelector)
-  const [modalInfoState, setModalInfoState] = React.useState(false)
+  const defaultTokenId = getDefaultAssetByNetworkId(wallet.networkId).identifier
+  const balance = useBalances(wallet, defaultTokenId)
```

Comment:
variable name and hook name should match, with few exceptions

Reply:
it was fixed in a previous commit. Thanks